app/src/main/java/net/gsantner/markor/format/general/CommonTextActions.java

```diff
@@ -245,7 +241,7 @@ protected void runIndentLines(Boolean deIndent) {
                 int spaceCount = textStart - lineStart;
                 int delCount = Math.min(_tabWidth, spaceCount);
                 int delEnd = lineStart + delCount;
-                if (delCount > 0 && delEnd < text.length()) {
+                if (delCount > 0 && delEnd <= text.length()) {
                     text.delete(lineStart, delEnd);
```

Comment:
potential out of index exception, can you confirm works fine on last line/ last character?

Reply:
I made this change because de-indent was not working on the last line. WIll test and confirm

Reply:
Looks like it works. I will be testing this for a day or so before I give the :+1:

Reply:
Good to go :+1: